src/webassembly/utils.rs

```diff
@@ -48,7 +48,7 @@ instance.data_pointers.globals \t- {:X} | offset - {:?}
         memories_pointer_address_0,
         0,
         memories_pointer_address_0_data,
-        memories_pointer_address_0_data - memories_pointer_address_0_data,
+        memories_pointer_address_0_data - instance_address,
```

Comment:
I assume this is `instance_address` to match the rest.

Reply:
Good call. Haven't realized of that typo